ProtocolTests/Protocol/QActions/QAction/CSharpCoreInterAppBrokerSupport/CSharpCoreInterAppBrokerSupport.cs

```diff
@@ -39,8 +52,22 @@ public void QAction_CSharpCoreInterAppBrokerSupport_Valid()
         #region Invalid Checks
 
         [TestMethod]
-#if !NET6_0_OR_GREATER
-        [Ignore]
+        [Ignore("Isn't really relevant and causes other checks to fail")]
+        public void QAction_CSharpCoreInterAppBrokerSupport_InvalidInterAppReplyLogic_XmlBased()
```

Comment:
Why add this if it gets ignored?

Reply:
Cause the management tool will generate the XMLs anyway and another unittest checks if the xml files are actually used in an unit test